stolon/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "stolon"
+version = "0.1.0"
+authors = ["Zcash Foundation <zebra@zfnd.org>"]
+license = "MIT OR Apache-2.0"
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+zebra-chain = { path = "../zebra-chain" }
+zebra-network = { path = "../zebra-network" }
+
+tokio = { version = "0.2.22", features = ["full"] }
+tokio-socks = { version = "0.3", features = ["tor"] }
```

Comment:
There are some rough spots in the `tokio-socks` crate (3rd party) but I think we could smooth them out and upstream some fixes.